src/spyglass/lfp/v1/lfp.py

```diff
@@ -164,15 +164,30 @@ def make(self, key):
                 "valid times",
             )
         )
-        IntervalList.insert1(
-            {
+
+        tmp_valid_times = (
+            IntervalList
+            & {
                 "nwb_file_name": key["nwb_file_name"],
                 "interval_list_name": key["interval_list_name"],
-                "valid_times": lfp_valid_times,
-                "pipeline": "lfp_v1",
-            },
-            replace=True,
-        )
+            }
+        ).fetch("valid_times")
+        if len(tmp_valid_times) == 0:
+            IntervalList.insert1(
+                {
+                    "nwb_file_name": key["nwb_file_name"],
+                    "interval_list_name": key["interval_list_name"],
+                    "valid_times": lfp_valid_times,
+                    "pipeline": "lfp_v1",
+                },
+                replace=True,
+            )
+        else:
+            assert np.isclose(
```

Comment:
- I was taught 'assert is only for testing', so the pattern should be `if not X: raise XError(msg)`. More about end-user transparency than functional changes
- `np.isclose` might be replaced by a hashing function. Not here, this is the perfect solve for a scope of 'fix bug in function', but I think it's maybe worth giving `IntervalList` a hashing method to check for collisions in `valid_times` - I plan to discuss with @edeno later today